modules/sdk-core/src/account-lib/mpc/tss/ecdsa/rangeproof.ts

```diff
@@ -253,7 +253,11 @@ export function verifyWithCheck(
   const modulusBytes = Math.floor((modulusBits + 7) / 8);
   const q = curve.order();
   const q3 = q ** BigInt(3);
-  if (proof.s1 == q3) {
+  const q7 = q ** BigInt(7);
+  if (proof.s1 > q3) {
```

Comment:
Just wanted to be sure that this should be gt and not gte, but lgtm

Reply:
The whitepaper allows for $s_1 <= q^3$ so gt is intentional here